libs/angular-integration-interface/src/lib/services/user.service.ts

```diff
@@ -40,7 +40,7 @@ export class UserService implements OnDestroy {
       ? this.permissions$.getValue()?.includes(permissionKey)
       : false
     const result = this.permissionsTopic$.getValue()
-      ? this.permissionsTopic$.getValue()?.includes(permissionKey)
+      ? this.permissionsTopic$.getValue()?.permissions?.includes(permissionKey)
```

Comment:
Here you need a more complex check because the permissions in the topic are not always for the current app